### PR TITLE
Edits to the Shuttle Static Folder discussion

### PR DESCRIPTION
**Pull request by sentinel1909**
I've done some editing to the Shuttle Static Folder entry under the Resources section of the Shuttle docs page.

I added a few details to embellish the explanation and story for the feature.

~Jeff Mitchell

### CODE REVIEW

**Comment by ivancernja** [comment]
Lgtm but would love second opinion from @oddgrd 

Thanks @sentinel1909

**Comment by sentinel1909** [comment]
> Lgtm but would love second opinion from @oddgrd
> 
> Thanks @sentinel1909

Sounds good, will watch for any further feedback :)

File: resources/shuttle-static-folder.mdx [hunk 1]
```diff
@@ -16,20 +16,29 @@ async fn main(
 ) -> __ { ... }
 ```
 
+Addition of the `shuttle_static_folder::StaticFolder` attribute will provide a `PathBuf` which has the location of the static folder. A `PathBuf` is an owned, mutable path (similar to a String). To know more about `PathBuf`, head over to the [discussion](https://doc.rust-lang.org/std/path/struct.PathBuf.html) in [The Rust Standard Library](https://doc.rust-lang.org/std/index.html). By default, the folder is called `static`, unless specifically renamed (see below).
+
+An example using the Axum framework can be found here: [Axum Static Files Example](https://github.com/shuttle-hq/examples/tree/main/axum/static-files)
```
Comment (oddgrd) [inline resources/shuttle-static-folder.mdx:21]:
```suggestion
An example using the Axum framework can be found here: [Axum Static Files Example](https://github.com/shuttle-hq/shuttle-examples/tree/main/axum/static-files)
```

**Review by oddgrd — APPROVED** [review]
LGTM too, thanks Jeff!

**Pull request merged**